security/try-node-to-node-encryption.md

```diff
@@ -43,7 +43,7 @@ Verify the operating system(s) running on the nodes in the cluster {% include op
     ```
    For OpenShift, add the Felix configuration with WireGuard enabled [under custom resources]({{ site.baseurl }}/getting-started/openshift/installation#optionally-provide-additional-configuration).    
 
-   > **Note**: This above command can be used to change other WireGuard attributes. For a list of other WireGuard parameters and configuration evaluation, see the [Felix configuration]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition).
+   >**Note**: This above command can be used to change other WireGuard attributes. For a list of other WireGuard parameters and configuration evaluation, see the [Felix configuration]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition).
```

Comment:
I don't know how sensitive jekyl/liquid is with the markdown here, but if I look at other examples that are rendering correctly, that don't have any spaces before the `>` and they have a space after the `>` before the `**note**`.

i.e. perhaps `  >**Notes**` might need to be `> **Notes**`